tests/nat/cli/test_plugin_loader.py

```
@@ -165,6 +165,33 @@ def test_load_click_group(self):
+        "finetune",
+        "info",
+        "object-store",
+        "optimize",
+        "red-team",
+        "registry",
+        "sizing",
+        "start",
+        "uninstall",
+        "validate",
+        "workflow",
+    }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)